### PR TITLE
docs: add README.md

### PR DESCRIPTION
**Pull request by elenacliu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,16 +56,62 @@ $ python finetune.py --model_name pointconv \
...
 $ python finetune.py --model_name pointconv \
     --train_dataset /path/to/unlabeled_data.bin \
     --interactive
 ```
 
+### Finetune pretrained-model with Pytorch Lightning
+#### prepare dataset
+
+To use your customized dataset, you should design your own dataset code, like those in `datasets/` directory. Here `datasets/modelnet40.py` is an example:
+
```
Comment (jemmyshin) [inline README.md:72]: tell users they need to implement `__len__()` and `__getitem__()`, if needed

File: README.md [hunk 2]
```diff
@@ -55,16 +56,62 @@ $ python finetune.py --model_name pointconv \
...
+    def __getitem__(self, index):
+        return (
+            # process on the fly, if needed
+            preprocess(self.points[index], num_points=self.sample_points),
+            self.labels[index],
+        )
+```
+
+#### finetuning model with labeled dataset
+
```
Comment (jemmyshin) [inline README.md:96]: where to find the supported model? Please list them

File: README.md [hunk 3]
```diff
@@ -55,16 +56,62 @@ $ python finetune.py --model_name pointconv \
...
+$ python finetune_pl.py --model_name pointconv \
+    --train_dataset /path/to/train.bin \
+    --eval_dataset /path/to/eval.bin \
+    --split_ratio 0.8 \
+    --checkpoint_path /path/to/checkpoint/ \
+    --embed_dim 512 \
+    --hidden_dim 1024 \
+    --batch_size 128 \
+    --epochs 50
+```
```
Comment (jemmyshin) [inline README.md:111]: we need benchmark for each model finally

File: README.md [hunk 4]
```diff
@@ -33,17 +33,18 @@ This Executor offers a GPU tag to speed up encoding. For more information on how
...
-### install finetuner
+### Finetune pretrained-model with finetuner
+#### install finetuner
 
 ```bash
 $ pip install finetuner
 ```
 
-### prepare dataset
+#### prepare dataset
```
Comment (jemmyshin) [inline README.md:43]: is this the same part as the `prepare dataset` in finetune part? If so, you can remove it.

**Review by jemmyshin — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (elenacliu) [reply README.md:43]: Here the original authors may want to get a dataset represented in jina's docarray. However when I used it, it reported bugs so I chose to use pytorchlighting instead of jina's finetuner.

File: README.md (hunk 3 above)
Comment (elenacliu) [reply README.md:111]: ok, I will add it after curvenet finishes its training
